apps/aecore/test/aecore_forking_SUITE.erl

```diff
@@ -120,18 +120,50 @@ sync_fork_in_wrong_order(Config) ->
     %% unexepctedly last block of dev1 arrives before rest of the chain
     rpc_call(N2, aec_conductor, post_block, [N1Top]), 
 
+    T0 = os:timestamp(),
     start_node_(dev1, Config),
     connect(N1),
-    timer:sleep(200),
+    await_sync_complete(T0, [N1, N2]),
```

Comment:
There is still a timing hole here, but it is hard to plug it. The chain_sync event might come before the subscription.

I think this is good enough.

Reply:
@ThomasArts yes, but this code should be refactored into a support library, since it's basically copy-pasted from the sync_SUITE.